src/main/java/com/minecolonies/coremod/colony/permissions/Permissions.java

```diff
@@ -833,11 +830,11 @@ public Rank getRank()
      */
     public static class View implements IPermissions
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this class from 195 to the maximum allowed 50 or externalize it in a public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2972)